scss/_variables.scss

```diff
@@ -1816,9 +1867,9 @@ $alert-dismissible-padding-r:       $alert-padding-y * 3 !default; // 3x covers
 // scss-docs-start progress-variables
 $progress-height:                   $spacer !default;
 $progress-font-size:                $font-size-base !default;
-$progress-bg:                       $gray-300 !default;
-$progress-border-radius:            $border-radius !default;
-$progress-box-shadow:               $box-shadow-inset !default;
+$progress-bg:                       $gray-300 !default; // Boosted mod: instead of `var(--#{$prefix}secondary-bg)`
```

Comment:
I think it could be done ?

Reply:
See https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/41495d7aa91997df6ff00ecf622dfa741a0f96dc.